sschipke-conga/black-stories-fe
# Show All Movies, Display Genres as Links on Movie Page, Display Trigger Warnings, Add Code of Conduct Page

## What's this PR do?
Show All Movies, Display Genres as Links on Movie Page, Display Trigger Warnings, Add Code of Conduct Page

### How should this be manually tested?
Try the *Show All*, try the *Code of Conduct Page*, test links on *Movie Page*, make sure movies with trigger warnings display them

### What are the relevant issues/tickets?
Fixes #30 #21 #10 #38 

### Build Preview!
https://60257ed0af49960008a11fcb--blackstories-dev.netlify.app/
